scripts/hypothesistooling.py

```diff
@@ -0,0 +1,79 @@
+import subprocess
+import os
+
+
+def current_branch():
+    return subprocess.check_output([
+        "git", "rev-parse", "--abbrev-ref", "HEAD"
+    ]).decode('ascii').strip()
+
+
+def tags():
+    result = [t.decode('ascii') for t in subprocess.check_output([
+        "git", "tag"
+    ]).split(b"\n")]
+    assert len(set(result)) == len(result)
+    return set(result)
+
+
+ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
this is a nasty failure if we ever move the scripts directory

`git rev-parse --show-toplevel` may be a better fit, and the following should be enough to get it

```
ROOT = subprocess.check_output(["git", "rev-parse",
    "--show-toplevel"]).decode('ascii').strip()
```